hooks/test_post_gen_project.py

```diff
@@ -116,16 +111,36 @@ def test_load_bib_info(
     shutil.copy(manuscript_path, manuscript_in_project)
     Path(project_dir_with_bib / "bib-in-project.bib").touch()
 
-    for file in missing_files:
-        Path(project_dir_with_bib / file).unlink()
-
-    for file in source_dir.iterdir():
-        if file.name in missing_files:
-            assert not (project_dir_with_bib / file.name).exists()
-        else:
-            assert (project_dir_with_bib / file.name).exists()
+    if not exists_bib:
+        for item in project_dir_with_bib.iterdir():
+            if item.isfile() and item.endswith(".bib"):
+                item.unlink()
 
-    load_bib_info(project_dir_with_bib, manuscript_in_project)
+    load_bib_info(project_dir_with_bib)
     actual_manuscript_content = manuscript_in_project.read_text()
     assert expected_manuscript_content == actual_manuscript_content
     project_dir_with_bib.unlink()
+<<<<<<< Updated upstream
+=======
+
+
+# C1: manuscript.tex doesn't exist in the project directory.
+#   Expect FileNotFoundError.
+def test_load_bib_info_bad(
+    user_filesystem, missing_files, expected_manuscript_content
+):
+
+    source_dir = user_filesystem / "user-repo-dir"
+    project_dir_with_bib = user_filesystem / "project-dir-with-bib"
+    shutil.copytree(source_dir, project_dir_with_bib)
```

Comment:
this can also be lighter-weight. I htink we just need the line ` project_dir_no_manuscript = user_filesystem / "project-dir-no-manuscript"`